framgia/ph_sels_rails_johnedel
# Create lessons for users

PR description: Create lessons for users
Content: Users can start a lesson
[Commands]
rails db:migrate:reset db:seed
rails db:server 

[Pre-conditions]
To test the module user need to login
go to categories and choose one lesson
users can answer lessons

to check more go to the database
there should be data in the lessons table and answers table associated to each other

[Expected Output]
Users can answer lessons created by the admin users. 
			
[Notes]
There is still no error traps where users can not answer, answered lessons
and there is still no result table.
